packages/ui/src/styles/globals.css

```
@@ -301,13 +301,81 @@
-    0 0 0 var(--surface-frame-halo-spread) var(--surface-frame-halo-color),
-    0 0 0 var(--surface-frame-ring-spread) var(--surface-frame-ring-color) !important;
+      var(--surface-frame-bevel-color) !important;
+
+  /* The ring, lining the reserved border from the border box inwards: outer
+   * edge on the element's own radius, inner edge on the padding box's, which
+   * is where the fill and the bevel end. The halo rides along because it needs
+   * the same exact box — its spacer is opaque, so the fraction of a pixel that
+   * ::after can be off by would paint the backdrop over the fill's corners and
+   * scrub the bevel out of them. Measured from the outer edge, which the ring
+   * no longer extends past, the halo gives back the ring's width. */
+  &::before {
```

Stylelint (17.14.0)

[error] Missing scoping root (nesting-selector-no-missing-scoping-root)

(nesting-selector-no-missing-scoping-root)